sdk/src/main/scala/io/horizen/fork/ActiveSlotCoefficientFork.scala

```diff
@@ -0,0 +1,18 @@
+package io.horizen.fork
+
+case class ActiveSlotCoefficientFork(
```

Comment:
Must the slot coefficient have a maximum ceiling value of 1 (100% of slots)?

Reply:
Yes good point.